dirty_plot/temp.csv

```diff
@@ -0,0 +1,2805 @@
+25.013521,-0.036992103

```

Comment:
Remove.
Please do check the staged changes before commit and do also check the PR  diff itself (either in code or on github).